moveit_ros/moveit_servo/test/servo_cpp_interface_test.cpp

```diff
@@ -59,6 +60,7 @@ class ServoFixture : public ::testing::Test
 public:
   void SetUp() override
   {
+    ros::Duration(ROS_SETUP_DELAY).sleep();
```

Comment:
It would be more robust to wait for certain topics/nodes to be available. Fixed timeouts can easily lead to flaky CI tests.

Reply:
I like the idea but (in practice) CI still fails. I'll add a check for a few key topics/parameters on top of the delay, though.

If you're curious, here's the CI run that failed without a startup delay.  https://github.com/ros-planning/moveit/runs/1210638771

Reply:
oh, it was a different issue. Done